lib/runners/processor.rb

```diff
@@ -218,15 +218,18 @@ def read_report_file(file_path = report_file)
     class InvalidXML < SystemError; end
 
     def read_xml(text)
-      root = REXML::Document.new(text).root
+      root = Nokogiri::XML(text) do |config|
+        config.options = Nokogiri::XML::ParseOptions::STRICT
```

Comment:
[suggestion] A simpler way is available as below:

```suggestion
        config.strict
```

https://nokogiri.org/tutorials/parsing_an_html_xml_document.html

Reply:
[question] What reason is set to `strict`, not default?

Reply:
the default is `RECOVER`, so every XML(include empty file) can be parsed even if the file is wrong like

```xml
<tag><xml></tag></xml>
```

Nokogiri returns `[<tag/><xml/>]`.

And some code expects invalid XML error, but never happen when I use the default.

Is it better to use the default(`RECOVER`)? 🤔

Reply:
OK, I see. Keep it as-is. 👍🏼

Reply:
fixed it on https://github.com/sider/runners/pull/2427/commits/f7bd1aaec4b91a9a69f1c77b06ab868e164d3583
Thanks